### PR TITLE
Use imageConfig in node

### PR DESCRIPTION
**Pull request by liggitt**
Make node and master config use identical imageConfig stanzas for ease of versioning

### CODE REVIEW

**Comment by liggitt** [comment]
@detiber something like this?

**Comment by detiber** [comment]
:+1:  lgtm

File: pkg/cmd/server/kubernetes/node_config.go [hunk 1]
```diff
@@ -84,10 +90,10 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 		ClusterDomain: options.DNSDomain,
 		ClusterDNS:    dnsIP,
 
-		VolumeDir:             options.VolumeDirectory,
-		NetworkContainerImage: options.NetworkContainerImage,
-		AllowDisabledDocker:   options.AllowDisabledDocker,
-		Client:                kubeClient,
+		VolumeDir:           options.VolumeDirectory,
+		ImageFor:            imageTemplate.ExpandOrDie,
```
Comment (deads2k) [inline pkg/cmd/server/kubernetes/node_config.go]: Why not expand to the name right here?  Does the node resolve other images?

**Comment by deads2k** [comment]
And I just got to the part in the email chain where Clayton answered that.  

ltgm [merge]

**Comment by deads2k** [comment]
@pweil-  router transient (https://ci.openshift.redhat.com/jenkins/job/merge_pull_requests_openshift3/1466/)

re[merge]

**Pull request merged**